.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+Your PR title must follow ![conventional commits](https://github.com/Uniswap/interface/pull/3832/files), and should start with one of the following ![types](https://github.com/Uniswap/interface/pull/3832/files):
```

Comment:
Update these links to point to conventional commits and the angular types (linked in slack)